irods/manager/data_object_manager.py

```diff
@@ -290,9 +290,17 @@ def open_with_FileRaw(self, *arg, **kw_options):
         handle = self.open(*arg,_raw_fd_holder=holder,**kw_options)
         return (handle, holder[-1])
 
+    _RESC_flags_for_open = frozenset((
+                               kw.RESC_NAME_KW,
+                               kw.DEST_RESC_NAME_KW,  # may be deprecated in the future
+                               kw.RESC_HIER_STR_KW,
```

Comment:
Extra comma here. Not sure if that's on purpose

Reply:
> Do we have a test which uses `DEST_RESC_HIER_STR_KW` (or any other unsupported keyword, for that matter) to demonstrate what happens when it is used with `open`? Does that add any value?

The parallel put test uses DEST_RESC_HIER_STR_KW.

Reply:
> Do we have a test which uses `DEST_RESC_HIER_STR_KW` (or any other unsupported keyword, for that matter) to demonstrate what happens when it is used with `open`? Does that add any value?

The present behavior is in the finest of PRC traditions, that is:  Let users apply the keywords they want to - and if the server ignores them silently,  the client will follows suit and the file ends up on the default resource.

Is that the behavior we want?  And do we care about its enforcement in the future?

Perhaps we'd prefer to chase down unfortunate keyword choices and throw exceptions when we see them.  If so, I expect we'd want to test for the success of trapping such a mistake, yes?

I guess the value of such a test depends on the resolution of these further questions.

Reply:
In light of those questions, I think adding such a test would not provide much value. Let's just leave it for now. Thanks!